coordinator/src/notification_service.rs

```diff
@@ -0,0 +1,132 @@
+use anyhow::Context;
+use anyhow::Result;
+use std::fmt::Display;
+use tokio::sync::mpsc;
+
+/// Types of notification that can be sent to 10101 app users
+
+#[derive(Debug, Clone)]
+pub enum NotificationKind {
+    /// Coordinator would like to settle the channel
+    ChannelClose,
+    PositionSoonToExpire,
+    PositionExpired,
+}
+
+impl Display for NotificationKind {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            NotificationKind::ChannelClose => write!(f, "ChannelClose"),
+            NotificationKind::PositionSoonToExpire => write!(f, "PositionSoonToExpire"),
```

Comment:
nit: maybe that's the `rollover` notification?

Reply:
I thought about it for a while, but the it's *before* rollover, and the user can decide whether to rollover or not.
I wonder whether we should also allow notifications *after* a successful rollover etc?